docs: Document DriverStationSettings.json fields (#42)
wpilibsuite/FirstDriverStation-Public: docs/Settings.md

### `UtilOpModeHash`

**Type**: integer (64-bit)

An internal identifier for the last selected Test/Utility game mode. This value is managed automatically by the DS and should not be edited manually.

### `MatchStartDelay`

**Type**: integer (seconds)

The delay in seconds before the match starts after the enable command is issued.

### `MatchAuto`

**Type**: integer (seconds)

The duration in seconds of the autonomous period in a match.

### `MatchTeleDelay`

**Type**: integer (seconds)

The delay in seconds between the end of the autonomous period and the start of the teleop period.

### `MatchTele`

**Type**: integer (seconds)

The duration in seconds of the teleop period in a match.

### `WindowMode`

**Type**: integer

Controls how the DS window is displayed. Valid values are:

| Value | Mode |
|---|---|
| `0` | Windowed (floating window) |
| `1` | Docked (attached to a screen edge) |

### `SelectedDashboard`

**Type**: integer

The dashboard application to launch alongside the DS. A value of `0` means no external dashboard is launched. All other values are assigned by the DS UI and are not defined here.

### `GameData`

**Type**: string

> A short string of game-specific data that is sent to the robot at the start of a match. The robot program can read this value to configure itself for a specific game variant. For details on how to read game data in your robot program, see the [WPILib documentation](https://docs.wpilib.org). The format and meaning of the game data string is defined by the game rules for each season.

Reviewer: ```suggestion
A short string of game-specific data that is sent to the robot during the match. The robot program can read this value to configure itself for a specific game variant. For details on how to read game data in your robot program, see the [WPILib documentation](https://docs.wpilib.org). The format and meaning of the game data string is defined by the game rules for each season.
```
Data is not always sent at the start of the match (e.g. 2026)
sorry for nitpicking haha

### `AutoOpModeHash`

**Type**: integer (64-bit)

An internal identifier for the last selected Autonomous game mode. This value is managed automatically by the DS and should not be edited manually.

### `UtilOpModeHash`

**Type**: integer (64-bit)

An internal identifier for the last selected Test/Utility game mode. This value is managed automatically by the DS and should not be edited manually.

### `MatchStartDelay`

**Type**: integer (seconds)

The delay in seconds before the match starts after the enable command is issued.

### `MatchAuto`

**Type**: integer (seconds)

The duration in seconds of the autonomous period in a match.

### `MatchTeleDelay`

**Type**: integer (seconds)

The delay in seconds between the end of the autonomous period and the start of the teleop period.

### `MatchTele`

**Type**: integer (seconds)

The duration in seconds of the teleop period in a match.

### `WindowMode`

**Type**: integer

Controls how the DS window is displayed. Valid values are:

| Value | Mode |
|---|---|
| `0` | Windowed (floating window) |
| `1` | Docked (attached to a screen edge) |

> ### `SelectedDashboard`

Reviewer: Pretty sure this section doesn't apply anymore?

Reviewer 2: SelectedDashboard does still apply. That is still in here. The other json file is just overrides to show more options.

Reviewer: Didn't  select the correct lines. I meant the values in the table below.

**Type**: integer

> The dashboard application to launch alongside the DS. Valid values are:

Reviewer 2: These values are not correct. We don't need to state in here what the valid values are. Just note that 0 is none, and any other values are UI dependent.

| Value | Dashboard |
|---|---|
| `0` | Default (no external dashboard) |
| `1` | SmartDashboard |
| `2` | Shuffleboard |
| `3` | FRC Dashboard |
| `4` | LabVIEW Dashboard |

### `GameData`

**Type**: string

> A short string of game-specific data that is sent to the robot at the start of a match. The robot program can read this value to configure itself for a specific game variant.

Reviewer: This should link to the docs page about the topic (https://docs.wpilib.org/en/stable/docs/yearly-overview/2026-game-data.html for 2026)

Reviewer 2: I don't want to point to a specific doc, because it changes every year, but I can add a note to look at the WPILib documentation and the game rules.

@copilot can you add a note to the game data section to tell users to look at the WPILib docs and the rules for specific information on game data?
